.gitignore

```diff
@@ -148,6 +148,7 @@ x64/
 x86/
 bld/
 [Ll]og/
+msbuild.binlog
```

Comment:
This is the binary log file that gets written when you run msbuild with flag `/bl` from the console. 
If you haven't used this, please install the viewer it from [https://msbuildlog.com/](https://msbuildlog.com/). You are inflicting too much suffering on yourself if you develop props/targets without it :)